demonstrations/tutorial_vqe.py

```diff
@@ -4,135 +4,108 @@
 
 .. meta::
     :property="og:description": Find the ground state of a Hamiltonian using the
-        variational quantum eigensolver algorithm in PennyLane.
+        variational quantum eigensolver algorithm.
     :property="og:image": https://pennylane.ai/qml/_images/pes_h2.png
 
 .. related::
 
+   tutorial_quantum_chemistry Building molecular Hamiltonians
    tutorial_vqe_parallel VQE with parallel QPUs
    tutorial_vqe_qng Accelerating VQE with the QNG
    tutorial_vqt Variational quantum thermalizer
 
-*Author: PennyLane dev team. Last updated: 8 Apr 2021.*
+*Author: PennyLane dev team. Last updated: 23 June 2021.*
 
-The Variational Quantum Eigensolver (VQE) [#peruzzo2014]_, [#yudong2019]_ is a flagship algorithm for
-quantum chemistry using near-term quantum computers. VQE is an application of the `Ritz variational
-principle <https://en.wikipedia.org/wiki/Ritz_method>`_  where a quantum computer is used to
-prepare a wave function ansatz of the molecule and estimate the expectation value of its electronic
-Hamiltonian while a classical optimizer is used to adjust the quantum circuit parameters in order
-to find the molecule's ground state energy.
+The Variational Quantum Eigensolver (VQE) [#peruzzo2014]_, [#yudong2019]_ is a flagship
+algorithm for quantum chemistry using near-term quantum computers. VQE is an application of the
+`Ritz variational principle <https://en.wikipedia.org/wiki/Ritz_method>`_ where a quantum
+computer is trained to prepare the ground state of a given molecule.
 
-For example, if we use a minimal basis, the ground state wave function of the hydrogen molecule
-:math:`\vert \Psi \rangle = \alpha \vert 1100 \rangle + \beta \vert 0011 \rangle` consists of only
-the Hartree-Fock component and a doubly-excited configuration where the two electrons occupy the
-highest-energy molecular orbitals. If we use a quantum computer to prepare the four-qubit
-entangled state :math:`\vert \Psi \rangle`, the ultimate goal of the VQE algorithm
-is to find the values of :math:`\alpha` and :math:`\beta` that minimize the expectation value of
-the electronic Hamiltonian.
+The inputs to the VQE algorithms are the molecular Hamiltonian, the
+parametrized circuit preparing the quantum state of the molecule, and the
+cost function to evaluate the expectation value of the Hamiltonian in the
```

Comment:
```suggestion
The inputs to the VQE algorithm are a molecular Hamiltonian, a
parametrized circuit preparing the quantum state of the molecule, and a
cost function to evaluate the expectation value of the Hamiltonian in the
```
For this sentence:
- "a" instead of "the" just to indicate that choices of these things are not unique
- for "inputs to the VQE algorithms", should algorithms be singular? There are many different variants of VQE, but since we are focusing on the "main" one.

Reply:
Good point. Perhaps the cost function is an input for more general VQAs, for the VQE not clear. I will modify this.